### PR TITLE
Add Linux/MinGW support and better error reporting

### LINKED ISSUE #1: 支持 Linux
通过检查 `uname` 的返回结果来判断是 Linux 还是 macOS，然后进一步决定是使用 `xdg-open` 还是 `open`。

### PR DESCRIPTION
**Pull request by ZMAlt**
close #1

### CODE REVIEW

File: gmtcn.sh [hunk 1]
```diff
@@ -50,30 +51,45 @@ if [ ! -n "${2}" ]; then
...
 fi
 
+# system
+sys=$(uname)
+if [ ${sys} = "Linux" ] ; then
+    open_command=xdg-open
+    ${open_command} --version > /dev/null || echo -e "\n\033[31m ERROR: ${open_command} does not exist. \033[0m"
+elif [ ${sys} = "Darwin" ] ; then
+    open_command=open
+elif [ ${sys} = *MIGW64* ] ; then
```
Comment (seisman) [inline gmtcn.sh]: MIGW64 or MINGW64?

File: gmtcn.sh [hunk 2]
```diff
@@ -50,30 +51,45 @@ if [ ! -n "${2}" ]; then
     exit
 fi
 
+# system
+sys=$(uname)
+if [ ${sys} = "Linux" ] ; then
+    open_command=xdg-open
```
Comment (seisman) [inline gmtcn.sh]: I think `open` should be good. No need to have such a long var name.
```suggestion
    open=xdg-open
```

File: gmtcn.sh (hunk 1 above)
Comment (ZMAlt) [reply gmtcn.sh]: MINGW64

File: gmtcn.sh (hunk 2 above)
Comment (ZMAlt) [reply gmtcn.sh]: updated

**Review by seisman — APPROVED** [review]

**Pull request merged**